tests/toDo.test.js

```diff
@@ -0,0 +1,72 @@
+import ToDo from "../src/models/toDo";
+import ToDoList from "../src/models/toDoList";
+import { describe, it, expect } from "vitest";
+
+describe("ToDo", () => {
+  it("should create a new ToDo", () => {
+    const form = document.createElement("form");
+    const input = document.createElement("input");
+    input.name = "description";
+    input.value = "Buy milk";
+    form.appendChild(input);
+
+    const toDo = new ToDo(form);
+
+    expect(toDo.id).toBeGreaterThan(0);
+    expect(toDo.description).toBe("Buy milk");
+    expect(toDo.done).toBe(false);
+    expect(toDo.createdAt).toBeInstanceOf(Date);
+  });
+
+  it("should update a ToDo", () => {
```

Comment:
```suggestion
  it("should update ToDo description properly", () => {
```

since we are updating description only.